src/ImageCarouselReact/widget/utils/bootstrapUtils.ts

```diff
@@ -10,18 +10,9 @@ export interface IBootstrapProps extends IObject {
   bsRole?: string;
 }
 
-/**
- * Returns a prefixed version of its parameters
- * i.e props.bsClass + "-" + variant
- *
- * @export
- * @param {IBootstrapProps} props
- * @param {string} [variant]
- * @returns
- */
-export function prefix(props: IBootstrapProps, variant?: string) {
-  if (props.bsClass === null) {
+export function prefix(bootstrapClass: string, variant?: string) {
```

Comment:
Please use in all file 4 spaces per tab indentation. 
Also on TransistionEvents.ts